node/overseer/overseer-gen/proc-macro/src/impl_builder.rs

```diff
@@ -130,7 +130,7 @@ pub(crate) fn impl_builder(info: &OverseerInfo) -> proc_macro2::TokenStream {
 				&mut self.handle
 			}
 			/// Obtain access to the overseer handle.
-			pub fn as_handle(&mut self) -> &#handle {
```

Comment:
Were these the causes of the warnings that would occur during macro expansion?

Reply:
This was just a borrow issue, the warnings with `+nightly` are partially resolved in #3874 , two remaining which are yet to be resolved.